administration/tests/test_admin_profiles.py

```diff
@@ -0,0 +1,93 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from administration.factories import (
+    AdminUserFactory,
+    AdminProfileFactory,
+)
+
+from utils.dump_response import dump  # noqa
+
+
+class TestAdminProfilesAPITests(APITestCase):
```

Comment:
I am sure we need tests for all BE functional of administration app views. Not only for get operations

Reply:
Done